src/content/docs/en/guides/cms/builderio.mdx

```diff
@@ -3,19 +3,507 @@ title: Builder.io & Astro
 description: Add content to your Astro project using Builder.io’s visual CMS
 type: cms
 service: Builder.io
-stub: true
+stub: false
 i18nReady: false
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro';
 
+[Builder.io](https://www.builder.io/) is a visual CMS that supports drag and drop content editing for building web sites.
 
+At the end of this recipe you'll be able to create blog posts in the Builder app and have your Astro site update with this new content automatically using a webhook.
+
+To do this, you'll use Builder's [Content API](https://www.builder.io/c/docs/content-api) and [Qwik API](https://www.builder.io/c/docs/qwik-api) to connect your Builder space to Astro with zero client-side JavaScript.
```

Comment:
Just checking, because you marked this as resolved but didn't accept the change and didn't try to negotiate keeping this.  :smile: I think all the above is uneccesary information at this stage of the instructions. It's better to keep information until the reader is at the point that they are using it. (Which is also my comment about the tip below)